packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -147,8 +175,10 @@ class _CodeViewState extends State<CodeView>
     if (parsedScript.lineCount * CodeView.rowHeight > extent) {
       // Scroll to the middle of the screen.
```

Comment:
this comment is outdated. Also, can we rename `centerScrollLocation` to `shouldCenterScrollLocation` so that it is more clearly a bool just by reading the code. When I first read this I assumed it would be a double